### PR TITLE
add context menu and sketch creation to homepage

### PR DESCRIPTION
**Pull request by Annoraaq**
image (4)

### CODE REVIEW

File: timesketch/frontend-v3/src/layouts/AppBar.vue [hunk 1]
```diff
@@ -29,17 +29,62 @@ limitations under the License.
...
+              prepend-icon="mdi-brightness-6"
+              v-on:click="toggleTheme"
+            >
+              <v-list-item-title>Toggle theme</v-list-item-title>
+            </v-list-item>
+            <v-list-item
+              href="/legacy/"
+              prepend-icon="mdi-view-dashboard-outline"
+            >
+              <v-list-item-title>Use the old UI</v-list-item-title>
```
Comment (berggren) [inline timesketch/frontend-v3/src/layouts/AppBar.vue:53]: I wonder if we should skip this for the new UI? @jkppr WDYT?

File: timesketch/frontend-v3/src/router/index.js [hunk 2]
```diff
@@ -12,6 +12,7 @@ import Default from "@/layouts/Default";
 
 // Import App views
 import Home from "@/views/Home.vue";
+import Overview from "@/views/Overview.vue";
```
Comment (berggren) [inline timesketch/frontend-v3/src/router/index.js:15]: What's the idea with the "Overview" page? :) Is this part of the "Sketch" view?

**Review by berggren — CHANGES_REQUESTED** [review]
Looks good, a couple of comments. Also, take a look at the conflicts and resolve / merge master branch.

File: timesketch/frontend-v3/src/layouts/AppBar.vue (hunk 1 above)
Comment (jkppr) [reply timesketch/frontend-v3/src/layouts/AppBar.vue:53]: Yes, I think with v3 we can finally deprecate the old legacy front-end.

Do we need a link to the frontend-ng folder though until we have everything ported?

File: timesketch/frontend-v3/src/router/index.js (hunk 2 above)
Comment (Annoraaq) [reply timesketch/frontend-v3/src/router/index.js:15]: I think I accidentally took that over from frontend-ng version:

```
 {
        path: '',
        name: 'Overview',
        redirect: { name: 'Explore' },
      },
```

I guess we don't need the Overview component and replace it with Canvas. I'll remove it from this CL as it is irrelevant for it anyway.

File: timesketch/frontend-v3/src/layouts/AppBar.vue (hunk 1 above)
Comment (Annoraaq) [reply timesketch/frontend-v3/src/layouts/AppBar.vue:53]: The route does not point anywhere yet, so we can still decide later what we mean by "old ui". I'd say we definitely don't need v1, but maybe ng is a good idea for the time of migration.

Comment (berggren) [reply timesketch/frontend-v3/src/layouts/AppBar.vue:53]: Sounds good

**Review by berggren — APPROVED** [review]

**Pull request merged**